library/test/src/formatters/junit.rs

```diff
@@ -56,7 +56,7 @@ impl<T: Write> OutputFormatter for JunitFormatter<T> {
         _state: &ConsoleTestState,
     ) -> io::Result<()> {
         // Because the testsuit node holds some of the information as attributes, we can't write it
```

Comment:
Given the typo correction below, I think these other two may be worth fixing at the same time?

```suggestion
        // Because the testsuite node holds some of the information as attributes, we can't write it
```